library/stats.py

```diff
@@ -428,7 +428,7 @@ def stats():
         if THEME_DATA['STATS']['MEMORY']['VIRTUAL']['USED'].get("SHOW", False):
             virtual_used = sensors.Memory.virtual_used()
 
-            virtual_used_text = f"{int(virtual_used / 1000000):>5}"
+            virtual_used_text = f"{int(virtual_used / 1000000)}"
```

Comment:
The padding in the value should not be removed: it is used to avoid "screen persistence" when a short value is written after a longer one.
![screencap](https://user-images.githubusercontent.com/38615348/213128949-1a4be79a-bf70-4445-b2f8-2e95c71303a3.png)
With the padding we are sure the longer value is erased completely, otherwise the previous unit is still displayed like in the picture.

Reply:
@mathoudebine just commited a fix, please try it again and see if it solves the problem

Reply:
I just tested your new modification, but there is still an issue with changing alignment of the virtual memory value from right to left.
It breaks other themes that are made for the value aligned to the right, like 3.5inchTheme:
![after-3 5inchTheme](https://user-images.githubusercontent.com/38615348/215153573-6f841a8f-25ac-4921-a6a8-a1227b80ba77.png)
or bash-dark-green:
![after-bash-dark-green](https://user-images.githubusercontent.com/38615348/215153613-9d4a5316-45d0-4f42-877f-a127eaa4a407.png)

I'm sorry but this modification cannot be made in the code for all themes.
I will have to think of a way to specify if the value must be left or right aligned in the theme file.

In the meantime, you can remove this modification from the PR so i can merge it here, and keep it only in your repository if you want!